module/mailnews/newsread.c

```diff
@@ -11,6 +11,9 @@
 
 #include "client.h"
 #include "mailnews.h"
+#include "string.h"
+
+#define MAX_HEADER_TEXT 256 // Named constant for buffer size
```

Comment:
This should be a static const int.

You say your editor is complaining?  It's possible it's treating this as a C language file (given its extension), while this is actually C++ code.  See if there's a setting for this.  Alternately we could rename every source file in the repo to .cpp, which probably will have to happen one day anyway.

Reply:
Aha, okay. I'm able to select the language mode for .c files. If we ever wanted to settle on Visual Studio Code as the IDE of choice, we could set up some project settings to default this.